koku/api/report/serializers.py

```diff
@@ -324,11 +324,13 @@ def validate(self, data):
         """
         super().validate(data)
 
-        if data.get("category") and ("key_only" not in data.keys()):
-            # allow category to be passed in without group by for the tags endpoint
-            if data.get("category") and (not data.get("group_by") or not data.get("group_by").get("project")):
-                error = {"error": ("Category may not be used without a group_by project parameter")}
-                raise serializers.ValidationError(error)
+        if data.get("category") and ("key_only" in data.keys()):
+            error = {"error": ("Category may only be used as a filter for the tag endpoints.")}
```

Comment:
Is `error` supposed to contain a single item tuple? If it's supposed to be a string, then the parenthesis aren't necessary.

Reply:
Thanks @samdoran that was bad copy & paste. 🙇

Reply:
I've been bitten many times by single item tuple bugs. 😄